samples/drivers/espi/src/main.c

```diff
@@ -31,7 +43,7 @@ static void espi_reset_handler(struct device *dev,
 			       struct espi_event event)
 {
 	if (event.evt_type == ESPI_BUS_RESET) {
-		printk("\neSPI BUS reset %d", event.evt_data);
+		LOG_INF("\neSPI BUS reset %d", event.evt_data);
```

Comment:
Remove all newlines (\n), they come with LOG_* for free. Also below in all changes from printk to LOG_*...